tools/nimbus-gradle-plugin/build.gradle

```diff
@@ -7,7 +7,6 @@ plugins {
     id 'groovy'
     id 'maven-publish'
     id 'groovy-gradle-plugin'
-    alias libs.plugins.kotlin.dsl
```

Comment:
Seems like this deletion doesn't hurt anything, but I don't have much context.  Tell me if it's needed.

Reply:
@jeddai Any thoughts on this? I'm inclined to agree that it can be removed if everything works without it, but you might have more context.

Reply:
I actually keep having to delete it locally because it won't resolve. Not sure when it got added but I don't see a problem with removing it